src/tests/system/tests/test_authentication.py

```diff
@@ -48,6 +48,144 @@ def test_authentication__with_default_settings(
     ), "User logged in with an invalid password!"
 
 
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+@pytest.mark.importance("critical")
+def test_authentication__change_password(
+    client: Client,
+    provider: GenericProvider,
+):
+    """
+    :title: User logins and issues a password change
+    :setup:
+        1. Create user 'user'
+        2. Start SSSD
+    :steps:
+        1. Login as user
+        2. Issue password change and enter a bad confirmation password
+        3. Issue password change and enter a good confirmation password
+        4. Login with old password
+        5. Login with new password
+    :expectedresults:
+        1. User is authenticated
+        2. Password is change is unsuccessful
+        3. Password change  is successful
+        4. User cannot log in
```

Comment:
I'm just curious if we have some "standard" terminology we should follow for cases like this around terminology like "authenticate" vs "log in" vs "login".   I tend to use "login" more than I think I should and have been trying to move to "authenticate" but, I have no problem with using "log in" and just wonder if we should pick one or just leave it to the author?

Reply:
We don't, and it may get too nitpicky, especially for non-native speakers.

log in (verb) - 'go ahead and log in'
login (noun) - 'enter your login' 
authenticate - you can authenticate without logging in, thinking about it, I favor login for these tests.

Reply:
Ok.   Fair enough.   I guess I was thinking about standardizing things maybe a little too generally.   Oh, and good point about authenticate.    closing this.